arrows/ocv/resection_camera.h

```diff
@@ -32,14 +32,10 @@ class KWIVER_ALGO_OCV_EXPORT resection_camera
   resection_camera();
   virtual ~resection_camera();
 
-  /// Get this algorithm's \link vital::config_block configuration block
```

Comment:
What is the reason for removing the doxygen comments here?  Does doxygen automatically pickup up the docs from the base class?

Reply:
it should, assuming INHERIT_DOCS=ON

Reply:
re-inserted the doxy